internal/indexmanagement/scripts.go

```diff
@@ -520,6 +565,7 @@ done
 
 const deleteThenRolloverScript = `
 set -uo pipefail
+source /tmp/scripts/indexManagement
```

Comment:
Is this source required? As per `/tmp/scripts/delete` already sourcing the same file?

Reply:
I believe this was added by mistake, my bad!